src/collectors/aws/redshift/index.ts

```diff
@@ -1,2 +1,8 @@
 export * from "./audit_logs";
 export * from "./clusters";
+export * from "./reserved_nodes";
```

Comment:
There is no file called "reserved_nodes". Getting error for this.

Reply:
The updated "index.ts" file has be uploaded, please have a look at it.